shell/app/modules/dcos/common/custom-label.tsx

```diff
@@ -114,3 +115,16 @@ export const checkCustomLabels = (_rule: any, value: string[], callback: Functio
     : false;
   return haveInCorrect ? callback(i18n.t('dcos:each label can only contain alphanumeric and underline')) : callback();
 };
+
+export const checkTagLabels = (_rule: any, value: string[], callback: Function) => {
+  const valueArr = [...isEmpty(value) ? [] : value];
+  const reg = /^[A-Za-z]([-A-Za-z0-9_.]+)[A-Za-z]$/;
+
+  const haveInCorrect = valueArr.length
+    ?
+    some(valueArr, (val: string) => {
+      return val.trim() ? !reg.test(val.trim()) : true;
+    })
+    : false;
+  return haveInCorrect ? callback(i18n.t('dcos:each label can only contain alphanumeric underscore underline and point')) : callback();
```

Comment:
prefer to rename 'haveInCorrect' to 'notPass';
`const notPass = valueArr.some(v => !reg.test(v));`

Reply:
error tip should add: start and end with alphabet, ...